ckanext/harvest/logic/action/get.py

```diff
@@ -361,17 +361,26 @@ def harvest_log_list(context, data_dict):
     return out
 
 
-def _get_sources_for_user(context, data_dict):
+def _get_sources_for_user(context, data_dict, harvest_source_list=None):
 
     session = context['session']
     user = context.get('user', '')
 
     only_active = data_dict.get('only_active', False)
     only_to_run = data_dict.get('only_to_run', False)
 
+    organization_id = request.params.get('organization_id')
+    if harvest_source_list:
+        limit = int(os.environ.get("HARVEST_SOURCE_LIST_LIMIT", "100"))
```

Comment:
Rather than using env vars, I think the convention in ckan is to get the setting from config object - https://github.com/alphagov/ckanext-harvest/blob/master/ckanext/harvest/queue.py#L39

Reply:
Okay sure